components/molecules/GameCard/GameCard.tsx

```diff
@@ -143,7 +144,7 @@ const GameCard: React.FC<GameCardProps> = (props) => {
                     borderColor: theme.colors.primarySand60,
                 },
                 name: {
-                    width: 120,
+                    width: 100,
```

Comment:
This will change the width of GameCard everywhere. Is that correct? Can you check with Saaniya and see if she intends to change *all* game card's to 100 x 100 or if we, instead, need to define a new variation for use in these feed stories?

Reply:
@krschacht Just checked with Saaniya this is one variation of game card in feed stories.

Reply:
@jeansampen Sorry, I don't follow you. Are you saying we are only using one variation of game card period—meaning, the only place we are using it is in the feed stories? Because I don't think that's the case. We also use GameCard on the Home screen of the app.

Or do you mean that the CSS you updated here only alters the variation of GameCard being used within the Feed story. If that's the case, then we should be good.

Reply:
Just meant it should be a variation for only game card but this code will affect all over. Let me update the width in variation of mini